sec-scanners-config.yaml

```diff
@@ -1,9 +1,9 @@
 module-name: istio
 protecode:
-  - europe-docker.pkg.dev/kyma-project/prod/istio-manager:1.3.0
-  - europe-docker.pkg.dev/kyma-project/prod/external/istio/install-cni:1.20.1-distroless
-  - europe-docker.pkg.dev/kyma-project/prod/external/istio/proxyv2:1.20.1-distroless
-  - europe-docker.pkg.dev/kyma-project/prod/external/istio/pilot:1.20.1-distroless
+  - europe-docker.pkg.dev/kyma-project/prod/istio-manager:1.3.1
```

Comment:
Istio still not released, this should be done with:
https://github.tools.sap/kyma/documentation/blob/main/kyma-internal/how-to-guides/istio-module/00-release-process.md#steps

Reply:
or if release is next, could you please update VERSION file or if we do not do it, adapt docs

Reply:
added the version file